crypto/s2n_stream_cipher_rc4.c

```diff
@@ -21,20 +21,30 @@
 #include "utils/s2n_blob.h"
 #include "utils/s2n_safety.h"
 
-static uint8_t s2n_stream_cipher_rc4_available()
+static const EVP_CIPHER *s2n_evp_rc4()
 {
 #ifdef S2N_LIBCRYPTO_SUPPORTS_EVP_RC4
+    return EVP_rc4();
```

Comment:
should this also check for Openssl-3?

Reply:
Nope. It's not testing for support, just limiting the blast radius of the #if we need to check for the existence of the EVP_rc4() symbol.